lib/Github/Api/Apps.php

```diff
@@ -137,16 +137,16 @@ public function removeInstallation($installationId)
      *
      * @link https://developer.github.com/v3/apps/installations/#list-repositories
      *
-     * @param int $userId
+     * @param array $params list of extra parameters.
      *
      * @return array
      */
-    public function listRepositories($userId = null)
+    public function listRepositories(array $params = [])
     {
-        $parameters = [];
-        if ($userId) {
-            $parameters['user_id'] = $userId;
-        }
+        $parameters = array_merge([
+            'page' => 1,
+            'per_page' => 30,
+        ], $params);
```

Comment:
This should not be needed as the [`ResultPager` class](https://github.com/KnpLabs/php-github-api/blob/aed01678b5a005b7430d8cb952985113177947fe/lib/Github/ResultPager.php) should be used to loop over multiple pages of results